src/dataDisplay/Table/table.stories.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react';
+
+import { Table, TableAlignment, TableSortDirection } from './index';
+import { Icon } from '../../';
+
+export default {
+  title: 'Data Display/Table',
+  component: Table,
+  parameters: {
+    componentSubtitle:
+      'Used for tabular information. It allows sorting by a single column.',
+  },
+};
+
+const headerCells = [
+  {
+    id: 'col1',
+    label: 'col1',
+  },
+  {
+    id: 'col2',
+    alignment: TableAlignment.right,
+    label: 'col2',
+  },
+  {
+    id: 'col3',
+    alignment: TableAlignment.right,
+    label: 'col3',
+  },
+];
+
+const rows = [
+  {
+    id: '1',
+    collapsibleContent: <div>content 1</div>,
+    cells: [
+      {
+        content: <Icon type="addressBook" size="sm" />,
+      },
+      {
+        content: 1,
+        alignment: TableAlignment.right,
+      },
+      {
+        content: 'safe',
+        alignment: TableAlignment.right,
+      },
+    ],
+  },
+  {
+    id: '2',
+    collapsibleContent: <div>content 2</div>,
+    cells: [
+      {
+        content: <Icon type="apps" size="sm" />,
+      },
+      {
+        content: 2,
+        alignment: TableAlignment.right,
+      },
+      {
+        content: 'gnosis',
+        alignment: TableAlignment.right,
+      },
+    ],
+  },
+];
+
+export const SimpleTable = (): React.ReactElement => (
+  <Table headers={headerCells} rows={rows} />
+);
+
+export const WithoutHeader = (): React.ReactElement => <Table rows={rows} />;
+
+export const Sortable = (): React.ReactElement => {
+  const [sortedByHeaderId, setSortedByHeaderId] = useState<string | undefined>(
+    'col2'
+  );
+  const [sortDirection, setSortDirection] = useState<TableSortDirection>(
+    TableSortDirection.asc
+  );
+
+  const onHeaderClick = (headerId: string) => {
+    if (!['col2'].includes(headerId)) {
+      return;
+    }
+
+    const newDirection =
+      sortedByHeaderId === headerId && sortDirection === TableSortDirection.asc
+        ? TableSortDirection.desc
+        : TableSortDirection.asc;
+
+    setSortDirection(newDirection);
+    setSortedByHeaderId(headerId);
+  };
+
+  const getSortedRows = () => {
+    const cp = [...rows];
+    return cp.sort((a, b) => {
+      return sortDirection === TableSortDirection.asc
```

Comment:
so this is sortable only by 1 header?

Reply:
yes, supporting multiple sorts it's a lot of work and safe-react only supports sorting by 1 header. It could be added later if we need to support more than 1.